app/backend/src/couchers/servicers/auth.py

```diff
@@ -154,11 +154,20 @@ def SignupFlow(self, request, context):
                     existing_user = session.execute(
                         select(User).where(User.email == request.basic.email)
                     ).scalar_one_or_none()
+                   
+
                     if existing_user:
                         context.abort(grpc.StatusCode.FAILED_PRECONDITION, errors.SIGNUP_FLOW_EMAIL_TAKEN)
                     existing_flow = session.execute(
                         select(SignupFlow).where(SignupFlow.email == request.basic.email)
                     ).scalar_one_or_none()
+                    
+                     existing_user_name = session.execute(
```

Comment:
There's an extra space on the beginning of this line, which should have been discovered when running the unit tests or running black.

```
couchers/app/backend$ black src/couchers/servicers/auth.py
error: cannot format src/couchers/servicers/auth.py: Cannot parse: 165:0:                      existing_user_name = session.execute(
Oh no! 💥 💔 💥
1 file failed to reformat.
```

Reply:
done
![Screenshot from 2021-08-19 20-01-33](https://user-images.githubusercontent.com/54991471/130087278-54bdca91-d619-4f24-82e1-365fd22a65a4.png)

Reply:
i am sorry for all the delay this is my first time contributing to opensource

Reply:
@kalvdans

Reply:
No problem :) I'm just a bit curious since you clicked in the "All test pass" and "Formatted my code" in the checklist above.

However, my `black` still wants to fix more style in your file, are you sure you have pushed the latest version?
```diff
(venv) chn@brahe:~/couchers/couchers/app/backend$ black src/couchers/servicers/auth.py
reformatted src/couchers/servicers/auth.py
All done! ✨ 🍰 ✨
1 file reformatted.
(venv) chn@brahe:~/couchers/couchers/app/backend$ git diff
diff --git a/app/backend/src/couchers/servicers/auth.py b/app/backend/src/couchers/servicers/auth.py
index 21e25bad..1082fb65 100644
--- a/app/backend/src/couchers/servicers/auth.py
+++ b/app/backend/src/couchers/servicers/auth.py
@@ -154,20 +154,19 @@ class Auth(auth_pb2_grpc.AuthServicer):
                     existing_user = session.execute(
                         select(User).where(User.email == request.basic.email)
                     ).scalar_one_or_none()
-                   
 
                     if existing_user:
                         context.abort(grpc.StatusCode.FAILED_PRECONDITION, errors.SIGNUP_FLOW_EMAIL_TAKEN)
                     existing_flow = session.execute(
                         select(SignupFlow).where(SignupFlow.email == request.basic.email)
                     ).scalar_one_or_none()
-                    
+
                     existing_user_name = session.execute(
                         select(User).where(User.username == request.basic.username)
                     ).scalar_one_or_none()
                     if existing_user_name:
                         context.abort(grpc.StatusCode.FAILED_PRECONDITION, errors.SIGNUP_FLOW_USERNAME_TAKEN)
-                        
+
                     if existing_flow:
                         send_signup_email(existing_flow)
                         session.commit()
```

Reply:
I only used black test. I will now use the test mentioned above...will that be all for formatting

Reply:
Sorry i edited the soace...but forgot to push the file...will do asap @kalvdans